requirements.txt

```diff
@@ -0,0 +1,141 @@
+aiohttp==3.8.3
```

Comment:
The Python dependencies that need to be installed for the OWL conversion, transitive dependencies fully resolved. See `requirements-unlocked.txt` for the base requirements (only 3 entries).

Reply:
I'm not sure where you are storing other Python requirements that need to be installed, but we should probably combine those requirements with these.